services/site/tests/components/homepage/challengeModal/ChallengeModalLevelCard.spec.tsx

```diff
@@ -14,6 +14,7 @@ describe("ChallengeModalLevelCard", () => {
         levelNumber={Number(mockLevelNumber)}
         status={LevelStatus.Open}
         isFirstUnfinishedLevel={false}
+        type={"CodingLevel"}
```

Comment:
Please add a test somewhere that checks if "CodingLevel" and "QuizLevel" lead to the expected outcome